okhttp/src/test/java/okhttp3/internal/ws/WebSocketReaderTest.java

```diff
@@ -113,12 +144,36 @@
     callback.assertTextMessage("Hello");
   }
 
+  @Test public void clientWithCompressionSimpleUncompressedHello() throws IOException {
+    data.write(ByteString.decodeHex("810548656c6c6f")); // Hello
+    clientReaderWithCompression.processNextFrame();
+    callback.assertTextMessage("Hello");
+  }
+
+  @Test public void clientWithCompressionSimpleCompressedHello() throws IOException {
+    data.write(ByteString.decodeHex("c107f248cdc9c90700")); // Hello
```

Comment:
oh sweet, that’s this one: https://tools.ietf.org/html/rfc7692#section-7.2.3.1
       0xf2 0x48 0xcd 0xc9 0xc9 0x07 0x00 0x00 0x00 0xff 0xff

Reply:
We should link that